src/pwr.ps1

```diff
@@ -721,8 +724,37 @@ function Exit-Shell {
 function Invoke-PwrScripts {
 	if ($PwrConfig.Scripts) {
 		$Name = $Run[0]
-		if ($PwrConfig.Scripts.$Name) {
-			$RunCmd = [String]::Format($PwrConfig.Scripts.$Name, [object[]]$Run)
+		$RunCmd = $PwrConfig.Scripts.$Name
+		if ($RunCmd) {
+			if ($RunCmd -is [PSCustomObject]) {
+				$FormatArgs = @($Name)
+				if ($RunCmd.Args -is [Array]) {
+					foreach ($R in $RunCmd) {
```

Comment:
I think you meant `$RunCmd.Args`?

Probably a good idea to add a test. This isn't working for me as-is.